### PR TITLE
Lt 920 finalised cases

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: templates/applications/tabs/case-notes.html [hunk 1]
```diff
@@ -6,20 +6,22 @@
...
-		</div>
+        {% if application.status.value|lower not in terminal_statuses%}
+		    <div class="lite-expandable-textarea lite-case-note-no-transition">
+			    <textarea id="case_note" name="text" cols="80" class="govuk-textarea" placeholder="Add note to application"></textarea>
+			    <div class="lite-expandable-textarea-controls">
+				    <p id="case_note-warning" class="govuk-hint govuk-character-count__message" aria-live="polite" style="text-align: left; margin: 2px 0 0 0;">
+					    {% get_string 'case_notes.max_length' %}
+				    </p>
+				    <a id="case-note-cancel-button" href="{% url 'applications:application' application.id 'case-notes' %}" class="govuk-button govuk-button--secondary govuk-button--secondary-white" type="button" draggable="false">
+					    Cancel
```
Comment (Mark-Spiers-kainos) [inline templates/applications/tabs/case-notes.html]: strings

File: ui_automation_tests/features/case_notes.feature [hunk 2]
```diff
@@ -30,3 +30,13 @@ Feature: I want to add a note to an application and view notes
...
+
+  @LT_920_prohibit_adding_case_note_in_terminal_status
+  Scenario: Unable to add case note
+    Given I go to exporter homepage and choose Test Org
+    And I create a standard application via api
+    When I click on applications
+    And my application has been withdrawn
+    And I click on application previously created
+    And I click the notes tab
+    Then the case note text area is not present
```
Comment (andytaylorkainos) [inline ui_automation_tests/features/case_notes.feature]: new line

Comment (stephanieboyd) [reply ui_automation_tests/features/case_notes.feature]: Black will just take it out agaaaaaaaaaain. Perhaps we can change it's configuration going forward but for now, the missing new line is needed to pass Circle CI checks :(

File: goods/views.py [hunk 3]
```diff
@@ -96,6 +97,11 @@ def get(self, request, **kwargs):
             "ecju_query_notifications": ecju_query_notifications,
         }
 
+        if self.good["query_id"]:
+            status_props, _ = get_status_properties(request, self.good["case_status"]["key"])
+            context["status_is_read_only"] = (status_props["is_read_only"],)
```
Comment (andytaylorkainos) [inline goods/views.py]: howcome this is in a tuple?

File: templates/goods/tabs/case-notes.html [hunk 4]
```diff
@@ -6,20 +6,22 @@
...
-		</div>
+        {% if not status_is_terminal %}
+		    <div class="lite-expandable-textarea lite-case-note-no-transition">
+			    <textarea id="case_note" name="text" cols="80" class="govuk-textarea" placeholder="Add note to application"></textarea>
+			    <div class="lite-expandable-textarea-controls">
+				    <p id="case_note-warning" class="govuk-hint govuk-character-count__message" aria-live="polite" style="text-align: left; margin: 2px 0 0 0;">
+					    {% get_string 'case_notes.max_length' %}
+				    </p>
+				    <a id="case-note-cancel-button" href="{{ post_url }}" class="govuk-button govuk-button--secondary govuk-button--secondary-white" type="button" draggable="false">
+					    Cancel
```
Comment (andytaylorkainos) [inline templates/goods/tabs/case-notes.html]: strings again :(

**Pull request merged**